.github/workflows/cron-correctness.yml

```diff
@@ -7,40 +7,6 @@ on:
   - cron: "0 * * * *"
 
 jobs:
-  bindingtester:
-    name: Binding Tester
-    runs-on: ubuntu-latest
+  docker:
 
-    strategy:
-      matrix:
-        parallelism: ["1", "2", "3"]
-    env:
-      CARGO_INCREMENTAL: 0
-      RUST_BACKTRACE: 1
-
-    steps:
-      - name: Checkout repository
-        uses: actions/checkout@v1
-        with:
-          submodules: recursive
-
-      - name: Install FoundationDB
-        uses: Clikengo/foundationdb-actions-install@v1
-
-      - name: Install Rust toolchain
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          profile: minimal
-
-      - name: Build
-        uses: actions-rs/cargo@v1
-        with:
-          command: build
-          args: -p bindingtester
-
-      - name: Setup bindingtester
-        run: scripts/setup_bindingtester.sh target/debug/bindingtester
-
-      - name: Run bindingtester
-        run: scripts/run_bindingtester.sh 300
+    uses: .github/workflows/pr-correctness.yml@main
```

Comment:
The script `scripts/run_bindingtester.sh` on both cron and PR files needs to be runned with different value:

* 300 for cron
* 1000 for pr

Can you add input parameters to the uses? It seems possible according to this [documentation](https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idstepswith). The parameter can be called `iterations`